workflow-dev/tpl/generate_params.toml

```diff
@@ -25,8 +25,10 @@ pullPolicy = "Always"
 dockerTag = "canary"
 
 [s3]
-accesskey = "YOUR KEY HERE"
-secretkey = "YOUR SECRET HERE"
+#Your aws access key.Leave it empty if you are on aws and want to use instance profile
```

Comment:
perhaps this would be better re-worded as

> ... if you are on AWS and want to use IAM credentials

Since more people understand what IAM is rather than "instance profile". also AWS should be capitalized